sympy/solvers/bivariate.py

```diff
@@ -508,3 +597,4 @@ def ok(f, v, c):
             if new is not None:
                 return a*x*y + b*y, new, u
             x, y = y, x
+from sympy.solvers.solveset import solvify
```

Comment:
Does this import need to be at the bottom because of circular imports? If so then that's fine. Otherwise it should go at the top of the file.

There should be two blank lines above the import line though.